File: app/src/main/java/com/kacper/and_krakgo/home/favourites/FavouritesFragment.java
package com.kacper.and_krakgo.home.favourites;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.kacper.and_krakgo.R;

/**
 * Created by kacper on 04/11/2017.
 */

public class FavouritesFragment extends Fragment{
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_favourites, container, false);
        return rootView;
    }
}

File: app/src/main/java/com/kacper/and_krakgo/screens/home/favourites/FavouritesFragment.kt
package com.kacper.and_krakgo.screens.home.favourites

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.kacper.and_krakgo.R

/**
 * Created by kacper on 04/11/2017.
 */

class FavouritesFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_favourites, container, false)
    }
}